adapters/richaudience/richaudience.go

```diff
@@ -0,0 +1,299 @@
+package richaudience
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+
+	"github.com/mxmCherry/openrtb/v15/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/metrics"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type adapter struct {
+	endpoint string
+}
+
+type richaudienceRequest struct {
```

Comment:
Is there a reason you're defining a new structure for the outgoing request instead of using the OpenRTB data model? The content seems to be a subset, but you're also dropping the US Privacy value.